cmd/kes/config.go

```diff
@@ -104,6 +104,11 @@ type serverConfig struct {
 				} `yaml:"tls"`
 			} `yaml:"keysecure"`
 		} `yaml:"gemalto"`
+
+		GCP struct {
```

Comment:
This configuration should follow the AWS secrets manager configuration:
```
GCP struct {
   SecretsManager struct {
       Endpoint string `yaml:"endpoint"`
       ...
   }
}
```

We want to also be able to support any GCP secretsmanager compatible implementation.